src/reminders/Reminder.ts

```diff
@@ -0,0 +1,77 @@
+import type { Moment } from 'moment';
+import { getSettings } from '../Config/Settings';
+
+export class ReminderSettings {
+    notificationTitle: string = 'Task Reminders';
+    dateFormat: string = 'YYYY-MM-DD';
+    dateTimeFormat: string = 'YYYY-MM-DD h:mm a';
+    dailyReminderTime: string = '9:00 am';
```

Comment:
For safety, I feel the saved reminder time should not be formatted. It should be `hh:mm` regardless of the user's format.

Reply:
Need some clarification on this. Do you mean keep all reminders when stored in cache in a single format (ISO maybe) that's independent of user format selection? If so, how would that work with creating Moment() from strings, don't you need to provide the input format?